wotw_seedgen/areas.wotw

```diff
@@ -12232,12 +12271,15 @@ anchor WestPools.Teleporter at -1656, -4171:  # the west Pools TP
       Sentry=2 OR Blaze=2
   pickup WestPools.BurrowEX:
     moki: Water, Burrow
-    kii: Damage=40, Burrow
-  pickup WestPools.BurrowOre:  # intended path omitted in moki
```

Comment:
This was intentional, some player can't collect this pickup even with all abilities available.
Logically unreachable pickups receive spirit light, so this is a good solution to ensure all moki players can complete their seeds.

Reply:
Sorry about that, I did not interpret the comment correctly. I removed the path and explained in the comment why it is omitted.